Header.c

```diff
@@ -20,51 +21,91 @@ in the source distribution for its full text.
 #include "XUtils.h"
 
 
-Header* Header_new(struct ProcessList_* pl, Settings* settings, int nrColumns) {
-   Header* this = xCalloc(1, sizeof(Header));
-   this->columns = xCalloc(nrColumns, sizeof(Vector*));
+Header* Header_new(struct ProcessList_* pl, Settings* settings, HeaderLayout hLayout) {
+   Header* this = xMalloc(sizeof(Header));
```

Comment:
Why not ensure zero-allocated structure with `xCalloc`?

Reply:
Yes, the xCalloc is better as the struct is not fully initialized in the lines below (pad, height).